api/dss.yaml

```diff
@@ -1119,7 +1191,9 @@ components:
           anyOf:
           - $ref: '#/components/schemas/IdentificationServiceArea'
           description: Resulting service area stored in DSS.
-    GetIdentificationServiceAreasResponse:
+        version:
+          $ref: '#/definitions/Version'
```

Comment:
This is not where Version is defined and therefore causes validation failure.

Reply:
fixed